vpr/src/base/clustered_netlist.h

```diff
@@ -228,6 +228,21 @@ class ClusteredNetlist : public Netlist<ClusterBlockId, ClusterPortId, ClusterPi
     ///@brief Sets the flag in net_is_global_ = state
     void set_net_is_global(ClusterNetId net_id, bool state);
 
+    /**
+     * @brief Given a name of a block and vector of possible cluster blocks
+     *        that are candidates to match the block name, go through 
+     *        the vector of cluster blocks and return the id of the block
+     *        where the block name matches the provided name.
+     * 
+     *        Given a string pattern representing a block name and a vector of
+     *        poissble cluster blocks that are candidates to match to the block
+     *        name pattern, go through the vector of cluster blocks and return 
+     *        the id of the block where the block name matches to the provided
+     *        input pattern.
+     * 
+     */
+    ClusterBlockId find_block_by_name_fragment(const std::string& name, const std::vector<ClusterBlockId>& cluster_block_candidates) const;
```

Comment:
Add @param to describe the inputs.
Change name to name_pattern

Reply:
Then can shorten description some (@param gives the input details).

Reply:
@return for return value

Reply:
Done